qdrant-landing/content/documentation/concepts/search.md

```diff
@@ -209,7 +209,33 @@ client.search(
 )
 ```
 
-Parameter `with_payload` might also be used to `include` or `exclude` specific fields only:
+The parameter `with_payload` may also be used to scope to or filter out a
+specific payload subset. You can specify an array of items to include. For
+example, to include `city`:
```

Comment:
I'd rewrite the whole page in [active voice](https://developers.google.com/style/voice) to tell make a reader think "This is what I can do". Here's how you can set up this paragraph with active voice:

```suggestion
You can use `with_payload` to scope to or filter a specific payload subset. 
You can even specify an array of items to include. For example, to include `city`, 
`village`, and `town`:
```